### DISCUSSION TITLE
2007/12/24/%E5%85%AC%E5%8F%B8%E5%B9%B4%E4%BC%9A

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#196] [Announcements]
# 2007/12/24/%E5%85%AC%E5%8F%B8%E5%B9%B4%E4%BC%9A

大约一个月前的某天，HR的头跑来把我叫到小会议室里对我说：“今年公司开年会（年末娱乐性质的），你来做主持吧”。我当时就有一种不祥的预感，如果有好事，她是绝对不会想到我的。不过鉴于她是负责发工资的，直接回绝恐怕不太好。犹豫了一下，我还是答应了，我说“反正我脸皮厚，随便你们捉弄就是了”。HR的头临走的时候神神秘秘的说：“你可要做好自毁形象的准备啊”。我突然感到后背发凉，天啊，以我对她的了解，她不是打算让我演“如花”吧。如果这样，我就宁死不从。

https://blog.qizhen.xyz/2007/12/24/%E5%85%AC%E5%8F%B8%E5%B9%B4%E4%BC%9A

**Comment by ruanqizhen** [comment]
之前博客下的留言：

竹贤
2008年01月10日 21:39 编辑
视频和照片打不开啊。

回复
Sammi
2008年01月2日 21:27 编辑
学习！！;0

回复
yoyo
2007年12月25日 19:46 编辑
有照片show一下就更形象了 

回复
ruirui
2007年12月25日 16:35 编辑
我也想看的说。。。

回复
斯
2007年12月25日 10:16 编辑
这么早就年会了。。。真洋气

回复
茂
2007年12月24日 18:02 编辑
有意思！

回复
